### PR TITLE
merge user settings with plugin settings + fix plugin install bug

### PR DESCRIPTION
**Pull request by barshimi**
Close: #7466 
Close: #7721 

## PR Checklist

- [ ] Tests for the changes have been added
- [ ] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

File: tutorials/amplication-intro.md
Comment (mulygottlieb) [inline tutorials/amplication-intro.md:1]: Not that I think this is important, but why did you delete this file as part of this PR?

Comment (barshimi) [reply tutorials/amplication-intro.md:1]: it was coming as part of the next merge

**Review by morhag90 — APPROVED** [review]

File: tutorials/amplication-intro.md
Comment (mulygottlieb) [reply tutorials/amplication-intro.md:1]: but on next this file exists

Comment (mulygottlieb) [reply tutorials/amplication-intro.md:1]: I added it back - resolving this conversation
